test/langtools/ProblemList.txt

```diff
@@ -48,9 +48,6 @@ tools/javac/annotations/typeAnnotations/referenceinfos/Lambda.java
 tools/javac/annotations/typeAnnotations/referenceinfos/NestedTypes.java         8057687    generic-all    emit correct byte code an attributes for type annotations
 tools/javac/warnings/suppress/TypeAnnotations.java                              8057683    generic-all    improve ordering of errors with type annotations
 tools/javac/modules/SourceInSymlinkTest.java                                    8180263    windows-all    fails when run on a subst drive
-tools/javac/processing/options/XprintRepeatingAnnotations.java                  8265611    generic-all    @compile/ref comparison fails when noSecurityManager=true
```

Comment:
Maybe close out JDK-8265611?

Reply:
Since you asked, I will close them as "cannot reproduce". :-)